fluent/EventsRune.ts

```diff
@@ -26,37 +28,3 @@ export interface FinalizationEventPhase {
 export interface InitializationEventPhase {
   type: "Initialization"
 }
-
-// TODO: delete this
-export type SystemExtrinsicFailedEvent = Event<{
```

Comment:
Moved this out of `EventsRune.ts` given that we only utilize within `ExtrinsicEventsRune.ts`

Reply:
I'll be cleaning this up further in a subsequent PR